netcdf_to_gltf_converter/config.py

```diff
@@ -108,11 +110,29 @@ class ShiftType(StrEnum):
 class CrsTransformation(BaseModel):
     """The configuration settings for transforming the coordinates."""
 
-    source_epsg: int
-    """int: EPSG code of the source coordinate system."""
+    source_crs: CRS
+    """CRS: The source coordinate system."""
 
-    target_epsg: int
-    """int: EPSG code of the target coordinate system."""
+    target_crs: CRS
+    """CRS: The target coordinate system."""
+
+    @validator("*", pre=True)
+    def validate_epsg(cls, value) -> CRS:
+        if isinstance(value, int):
+            return create_crs(value)
+
+        if isinstance(value, str):
+            return CrsTransformation._create_crs_from_string(value)
+
+        return value
```

Comment:
Is it safe to just return value here? since the return is typehinted on CRS, is value always CRS when it is not an int or str?

Reply:
This will give a validation error: if it cannot be converted to a CRS:

![image](https://github.com/Deltares-research/netcdf_to_gltf_converter/assets/36264671/2d47cbfc-c9e7-43f4-9242-9b296aed7cdb)

This is wanted behaviour.